src/main/java/core/basesyntax/Application.java

```diff
@@ -0,0 +1,29 @@
+package core.basesyntax;
+
+public class Application {
+    public static void main(String[] args) {

```

Comment:
Don't begin method implementation with an empty line. Remove the redundant empty line at the beginning of the `main` method.